test/inductor/test_torchinductor.py

```diff
@@ -3256,16 +3256,14 @@ def fn(x):
         )
 
     def test_avg_pool2d7(self):
-        # Large kernel size, use fallback
+        # Large kernel size
         def fn(x):
             return aten.avg_pool2d(x, [13, 13], [1, 1], [0, 0])
 
-        torch._inductor.metrics.generated_kernel_count = 0
         self.common(
             fn,
             (-torch.arange(1 * 24 * 24, dtype=torch.float32).view(1, 1, 24, 24),),
         )
-        self.assertEqual(torch._inductor.metrics.generated_kernel_count, 0)
```

Comment:
This was checking no inductor kernel was generated for large kernel sizes (window size >25) since the lowering fallbacks to eager mode for that case. The decomposition works for any window size and therefore this test is wrong now.

Reply:
Note that the fallback was there not for correctness but for performance. We should find what are relevant shapes to test this operation on and make sure that we are not regressing on those.